lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -26,8 +26,7 @@ updateScriptTags = (activeDocument, newScripts, callback) ->
   )
 
 extractTrackedAssets = (doc) ->
-  for node in doc.head.children when node.dataset.turbolinksTrack?
-    node
+  [].slice.call(doc.querySelectorAll('[data-turbolinks-track]'))
```

Comment:
The performance diffs here are so negligible it doesn't really matter what you choose unless you're going to run this hundreds of thousands of times.

Average of running 1 million instances. Each one of these were run 5 separate times.

| [].slice.call | Array.prototype.slice.call |
| ------- |------------------
| 394.88ms |378.16ms |

That's about 16ms at 1 million times, about 1 frame.